File: Resueltos/Cristian Maldonado/FizzBuzz.cpp
#include<iostream>
#include<bits/stdc++.h>
using namespace std;

int main (int argc, char *argv[]) {
	int casos  = 0;
	scanf("%d",&casos);
	for (int i = 0; i < casos ; i++){
		int divisor1 = 0; 
		int divisor2 =0;
		int numero = 0;
		scanf("%i",&divisor1);
		scanf("%i",&divisor2);	
		scanf("%i",&numero);
		for(int j = 1; j <=numero; j++){
			if(j%divisor1 == 0 && j % divisor2 ==0){
				cout << "FizzBuzz"<<endl;
			}else if(j % divisor1 ==0){
				cout << "Fizz"<<endl;
			}else if(j % divisor2 == 0){
				cout << "Buzz"<<endl;
			}else{
				cout << j << endl;
			}
		}
	
	}
	return 0;
}

File: Resueltos/Luciano_Chavarria/Python/FizzBuzz.py
for j in range(int(input())):
    x,y,n = input().split()
    x,y,n = int(x),int(y),int(n)
    for i in range(1,n+1):
        if i%x==0 and i%y==0:
            print("FizzBuzz")
            continue
        if i%x==0:
            print("Fizz")
            continue
        if i%y==0:
            print("Buzz")
            continue
        else:
            print(i)
